SimCalorimetry/HGCalSimProducers/plugins/HGCDigitizer.cc

```diff
@@ -247,7 +249,7 @@ HGCDigitizer::HGCDigitizer(const edm::ParameterSet& ps, edm::ConsumesCollector&
   premixStage1_ = ps.getParameter<bool>("premixStage1");
   premixStage1MinCharge_ = ps.getParameter<double>("premixStage1MinCharge");
   premixStage1MaxCharge_ = ps.getParameter<double>("premixStage1MaxCharge");
-  iC.consumes<std::vector<PCaloHit>>(edm::InputTag("g4SimHits", hitCollection_));
```

Comment:
@bsunanda , is it a good idea to remove "consumes"?

Reply:
it has been moved to hitToken_ - here https://github.com/cms-sw/cmssw/pull/37856/files#diff-563eece7ae99b29c2501955e163a955bd6940d6fa39094e919eee33975bdb1f2R235

Reply:
As Pedro says, it has been taken care of by moving this in defining the token